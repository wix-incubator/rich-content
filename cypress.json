{
  "chromeWebSecurity": false,
  "baseUrl": "http://localhost:3002",
  "testFiles": "**/*.e2e.js",
  "ignoreTestFiles": [
    "**/__snapshots__/*",
    "**/__image_snapshots__/*"
  ],
  "integrationFolder": "e2e",
  "fixturesFolder": "e2e/cypress/fixtures",
  "pluginsFile": "e2e/cypress/plugins/index.js",
  "screenshotsFolder": "e2e/cypress/screenshots",
  "supportFile": "e2e/cypress/support/index.js",
  "video": true,
  "videoUploadOnPasses": false,
  "videosFolder": "e2e/cypress/videos",
  "env": {},
  "projectId": "6yj6sr"
}
