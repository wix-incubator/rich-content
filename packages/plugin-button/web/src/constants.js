/* eslint-disable camelcase */
export const BUTTON_TYPE = 'wix-draft-plugin-button';
export const ALIGN_CENTER = 'center';
export const settingsTabValue = 'settings';
export const designTabValue = 'design';
export const BUTTON_TYPES = ['primary', 'secondary'];
export const DEFAULT_COLOR = '#000000';
export const DEFAULT_SELECTION_COLOR = '#000000';
export const COLOR_PICKER_TYPE = Object.freeze({
  TEXT_COLOR: 'textColor',
  BORDER_COLOR: 'borderColor',
  BACKGROUND_COLOR: 'backgroundColor',
});

export const COLORS = Object.freeze({
  color1: '#FFFFFF',
  color2: '#D5D4D4',
  color3: '#000000',
  color4: '#000000',
  color5: '#000000',
  color6: '#ABCAFF',
  color7: '#81B0FF',
  color8: '#0261FF',
  color9: '#0141AA',
  color10: '#012055',
});

export const DEFAULT_CONFIG = {
  alignment: ALIGN_CENTER,
  size: 'small',
  width: 'fit-content',
};

export const buttonPreviews = colors => [
  {
    className: 'button_primary',
    border: '0px solid #' + colors.color8,
    borderRadius: '0px',
    borderWidth: '0px',
    background: colors.color8,
    color: colors.color1,
    borderColor: colors.color8,
  },
  {
    className: 'button_secondary',
    border: '1px solid ' + colors.color8,
    borderRadius: '0px',
    borderWidth: '1px',
    background: colors.color1,
    color: colors.color8,
    borderColor: colors.color8,
  },
  {
    className: 'button_secondary',
    border: '1px solid ' + colors.color8,
    borderRadius: '0px',
    borderWidth: '1px',
    background: colors.color7,
    color: colors.color8,
    borderColor: colors.color8,
  },
  {
    className: 'button_secondary',
    border: '1px solid ' + colors.color8,
    borderRadius: '10px',
    borderWidth: '1px',
    background: colors.color7,
    color: colors.color8,
    borderColor: colors.color8,
  },
  {
    className: 'button_secondary',
    border: '1px solid ' + colors.color8,
    borderWidth: '5px',
    borderRadius: '0px',
    background: colors.color1,
    color: colors.color8,
    borderColor: colors.color8,
  },
];
