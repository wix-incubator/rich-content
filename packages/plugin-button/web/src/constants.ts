/* eslint-disable camelcase */
export const ALIGN_CENTER = 'center';
export const settingsTabValue = 'settings';
export const designTabValue = 'design';
export const BUTTON_TYPES = ['primary', 'secondary'];
export const DEFAULT_COLOR = '#000000';
export const DEFAULT_SELECTION_COLOR = '#000000';
export const COLOR_PICKER_TYPE = Object.freeze({
  TEXT_COLOR: 'textColor',
  BORDER_COLOR: 'borderColor',
  BACKGROUND_COLOR: 'backgroundColor',
});

export const DEFAULT_CONFIG = {
  alignment: ALIGN_CENTER,
  size: 'small',
  width: 'fit-content',
};
