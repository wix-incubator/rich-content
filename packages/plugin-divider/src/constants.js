export const DIVIDER_TYPE = 'wix-draft-plugin-divider';

export const LINE_SINGLE = 'single';
export const LINE_DOUBLE = 'double';
export const LINE_DASHED = 'dashed';
export const LINE_DOTTED = 'dotted';

export const LINE_TYPES = [LINE_SINGLE, LINE_DOUBLE, LINE_DASHED, LINE_DOTTED];

export const ALIGN_LEFT = 'left';
export const ALIGN_CENTER = 'center';
export const ALIGN_RIGHT = 'right';

export const SIZE_SMALL = 'small';
export const SIZE_MEDIUM = 'medium';
export const SIZE_LARGE = 'large';

export const DEFAULTS = {
  type: LINE_SINGLE,
  config: {
    dividerSize: SIZE_LARGE,
    dividerAlignment: ALIGN_CENTER
  }
};
