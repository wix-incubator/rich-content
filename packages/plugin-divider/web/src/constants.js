export const DIVIDER_TYPE = 'wix-draft-plugin-divider';

export const LINE_SINGLE = 'single';
export const LINE_DOUBLE = 'double';
export const LINE_DASHED = 'dashed';
export const LINE_DOTTED = 'dotted';

export const LINE_TYPES = Object.freeze([LINE_SINGLE, LINE_DOUBLE, LINE_DASHED, LINE_DOTTED]);

export const ALIGN_LEFT = 'left';
export const ALIGN_CENTER = 'center';
export const ALIGN_RIGHT = 'right';

export const SIZE_SMALL = 'small';
export const SIZE_MEDIUM = 'medium';
export const SIZE_LARGE = 'large';

export const SIZES = Object.freeze([SIZE_LARGE, SIZE_MEDIUM, SIZE_SMALL]);
export const ALIGNMENTS = [ALIGN_LEFT, ALIGN_CENTER, ALIGN_RIGHT];
export const NO_WRAP = 'nowrap';

export const DEFAULTS = Object.freeze({
  type: LINE_SINGLE,
  config: {
    size: SIZE_LARGE,
    alignment: ALIGN_CENTER,
    textWrap: NO_WRAP,
  },
});
