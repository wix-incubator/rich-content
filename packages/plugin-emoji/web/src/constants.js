import React from 'react';
import FaBell from './icons/FaBell.svg';
import FaFlag from './icons/FaFlag.svg';
import FaHeart from './icons/FaHeart.svg';
import FaPaw from './icons/FaPaw.svg';
import FaSmile from './icons/FaSmile.svg';
import FaPlane from './icons/FaPlane.svg';
import FaUtensils from './icons/FaUtensils.svg';
import FaRegFutbol from './icons/FaRegFutbol.svg';
export const EMOJI_TYPE = 'wix-draft-plugin-emoji';

export const getEmojiGroups = t => [
  {
    title: t('EmojiPlugin_EmojiGroups_People'),
    icon: <FaSmile />,
    category: 'people',
  },
  {
    title: t('EmojiPlugin_EmojiGroups_Nature'),
    icon: <FaPaw />,
    category: 'nature',
  },
  {
    title: t('EmojiPlugin_EmojiGroups_Food'),
    icon: <FaUtensils />,
    category: 'food',
  },
  {
    title: t('EmojiPlugin_EmojiGroups_Activity'),
    icon: <FaRegFutbol />,
    category: 'activity',
  },
  {
    title: t('EmojiPlugin_EmojiGroups_Travel'),
    icon: <FaPlane />,
    category: 'travel',
  },
  {
    title: t('EmojiPlugin_EmojiGroups_Objects'),
    icon: <FaBell />,
    category: 'objects',
  },
  {
    title: t('EmojiPlugin_EmojiGroups_Symbols'),
    icon: <FaHeart />,
    category: 'symbols',
  },
  {
    title: t('EmojiPlugin_EmojiGroups_Flags'),
    icon: <FaFlag />,
    category: 'flags',
  },
];

export const DesktopFlyOutModalStyles = Object.freeze({
  overlay: {
    backgroundColor: 'rgba(0, 0, 0, 0)',
    zIndex: 5,
  },
  content: {
    width: '320px',
    boxSizing: 'border-box',
    height: '364px',
    overflow: 'visible',
    border: 'solid 1px rgba(51, 51, 51, 0.1)',
    paddingRight: '0px',
    paddingLeft: '0px',
    display: 'block',
    borderRadius: '2px',
    position: 'absolute',
    zIndex: 6,
    paddingTop: '20px',
    boxShadow: '0 0 10px 0 rgba(0, 0, 0, 0.06)',
  },
});

export const DEFAULT_CONFIG = {};
