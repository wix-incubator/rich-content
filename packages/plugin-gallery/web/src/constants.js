const GALLERY_LAYOUTS = {
  EMPTY: -1,
  COLLAGE: 0,
  MASONRY: 1,
  GRID: 2,
  THUMBNAIL: 3,
  SLIDER: 4,
  SLIDESHOW: 5,
  PANORAMA: 6,
  COLUMN: 7,
  MAGIC: 8,
  FULLSIZE: 9,
  BRICKS: 10,
  MIX: 11,
  ALTERNATE: 12,
};

const HORIZONTAL_LAYOUTS = [
  GALLERY_LAYOUTS.THUMBNAIL,
  GALLERY_LAYOUTS.SLIDER,
  GALLERY_LAYOUTS.SLIDESHOW,
  GALLERY_LAYOUTS.COLUMN,
  GALLERY_LAYOUTS.FULLSIZE,
];

export const sampleItems = [1, 2, 3].map(i => {
  return {
    metadata: {
      height: 10,
      width: 10,
    },
    orderIndex: i,
    itemId: 'sampleItem-' + i,
    url:
      'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAALEwAACxMBAJqcGAAAAVlpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IlhNUCBDb3JlIDUuNC4wIj4KICAgPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4KICAgICAgPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIKICAgICAgICAgICAgeG1sbnM6dGlmZj0iaHR0cDovL25zLmFkb2JlLmNvbS90aWZmLzEuMC8iPgogICAgICAgICA8dGlmZjpPcmllbnRhdGlvbj4xPC90aWZmOk9yaWVudGF0aW9uPgogICAgICA8L3JkZjpEZXNjcmlwdGlvbj4KICAgPC9yZGY6UkRGPgo8L3g6eG1wbWV0YT4KTMInWQAAAA1JREFUCB1jePv27X8ACVkDxyMHIvwAAAAASUVORK5CYII=', //eslint-disable-line
  };
});

export const getDefault = () => ({
  items: [],
  styles: {
    galleryLayout: 2,
    gallerySizeType: 'px',
    gallerySizePx: 300,
    galleryMargin: 0,
    oneRow: false,
    cubeRatio: 1,
    galleryThumbnailsAlignment: 'bottom',
    isVertical: false,
    imageMargin: 20,
    thumbnailSpacings: 0,
    cubeType: 'fill',
    enableInfiniteScroll: true,
    titlePlacement: 'SHOW_ON_HOVER',
    allowHover: false,
    itemClick: 'link',
    fullscreen: false,
    showArrows: false,
    gridStyle: 1,
    loveButton: false,
    allowSocial: false,
    allowDownload: false,
    mobileSwipeAnimation: 'NO_EFFECT',
    thumbnailSize: 120,
    gotStyleParams: true,
  },
  config: {
    alignment: 'center',
    size: 'content',
    layout: 'small',
    spacing: 0,
  },
});

export const isHorizontalLayout = ({ galleryLayout }) =>
  HORIZONTAL_LAYOUTS.indexOf(galleryLayout) > -1;
