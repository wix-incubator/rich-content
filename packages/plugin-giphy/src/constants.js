export const PAGE_SIZE = 25;
export const WAIT_INTERVAL = 300;
export const SEARCH_TYPE = 'gifs';
export const MODAL_TYPE = 'flyOutModal';
export const GIPHY_TYPE = 'wix-draft-plugin-giphy';
export const GIPHY_API_KEY = 'wv9cEt7t1nu5wFYinAs73zwDzrrclGqk';

const gphApiClient = require('giphy-js-sdk-core');
export const giphyApiClient = gphApiClient(GIPHY_API_KEY);

export const DEFAULTS = {
  config: {
    size: 'content',
    alignment: 'center'
  },
};

export const MobileFullScreenCustomStyle = {
  overlay: {
    backgroundColor: 'transparent'
  },
  content: {
    top: 0,
    left: 0,
    overflow: 'hidden',
    paddingRight: '6px'
  }
};

export const DesktopFlyOutModalStyles = {
  overlay: {
    backgroundColor: 'rgba(0, 0, 0, 0)',
    zIndex: 10
  },
  content: {
    width: '265px',
    boxSizing: 'border-box',
    height: '357px',
    overflow: 'visible',
    border: '1px solid #ccc',
    paddingRight: '8px',
    paddingLeft: '18px',
    display: 'block',
    position: 'absolute',
    zIndex: 6,
    paddingTop: '8px'
  }
};
