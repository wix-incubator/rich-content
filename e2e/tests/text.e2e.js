import { INLINE_TOOLBAR_BUTTONS } from '../cypress/dataHooks';
import { DEFAULT_DESKTOP_BROWSERS } from './constants';

describe('text', () => {
  before(function() {
    cy.eyesOpen({
      appName: 'Text',
      testName: this.test.parent.title,
      browser: DEFAULT_DESKTOP_BROWSERS,
    });
  });

  beforeEach(() => cy.switchToDesktop());

  afterEach(() => cy.matchContentSnapshot());

  after(() => cy.eyesClose());

  it('allow to enter text', function() {
    cy.loadEditor()
      .enterParagraphs([
        'Leverage agile frameworks',
        'to provide a robust synopsis for high level overviews.',
      ])
      .setSelection(0, 0)
      .blurEditor();
    cy.eyesCheckWindow(this.test.title);
  });

  it('allow to apply inline styles and links', function() {
    cy.loadEditor('plain')
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.BOLD, [40, 10])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.UNDERLINE, [10, 5])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.ITALIC, [20, 5])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.BOLD, [30, 5])
      .setLineSpacing(1, [10, 50])
      .setColor(4, [200, 208])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.UNDERLINE)
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.ITALIC)
      .setAlignment(INLINE_TOOLBAR_BUTTONS.TEXT_ALIGN_CENTER)
      .setAlignment(INLINE_TOOLBAR_BUTTONS.TEXT_ALIGN_RIGHT)
      .setAlignment(INLINE_TOOLBAR_BUTTONS.TEXT_ALIGN_LEFT)
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.QUOTE, [30, 170])
      .setLink([0, 10], 'https://www.wix.com/')
      .setLink([50, 65], 'https://www.one.co.il/')
      .setColor(1, [300, 305])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.TITLE, [250, 260])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.QUOTE, [250, 260])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.ORDERED_LIST)
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.UNORDERED_LIST)
      .setLineSpacing(3, [100, 150])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.CODE_BLOCK, [100, 300])
      .setLink([15, 30], 'https://www.sport5.co.il/')
      .setSelection(0, 0)
      .enterParagraphs(['#LIVING THE DREAM\n'])
      .setLink([0, 17], 'https://www.sport5.co.il')
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.CODE_BLOCK, [0, 10])
      .setSelection(0, 0)
      // TODO: should fix unstable behavior of mention
      // .enterParagraphs(['@NO_MORE\n'])
      .setLink([0, 10], 'https://www.wix.com/')
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.CODE_BLOCK, [0, 10])
      .blurEditor();
    cy.eyesCheckWindow(this.test.title);
  });

  it('allow to create lists', function() {
    cy.loadEditor('plain')
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.ORDERED_LIST, [300, 100])
      .setTextStyle(INLINE_TOOLBAR_BUTTONS.UNORDERED_LIST, [550, 1]);
    cy.eyesCheckWindow(this.test.title);
  });
});
