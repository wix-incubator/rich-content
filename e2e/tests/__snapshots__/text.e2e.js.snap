exports[`text > allow to enter text #0`] =
{
  "blocks": [
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 0,
      "text": "Leverage agile frameworks",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 1,
      "text": "to provide a robust synopsis for high level overviews.",
      "type": "unstyled"
    }
  ],
  "entityMap": {}
};

exports[`text > allow to apply inline styles and links #0`] =
{
  "blocks": [
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 0,
      "text": "",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 1,
      "text": "",
      "type": "unstyled"
    },
    {
      "data": {
        "textAlignment": "left"
      },
      "depth": 0,
      "entityRanges": [
        {
          "key": 0,
          "length": 10,
          "offset": 0
        },
        {
          "key": 1,
          "length": 7,
          "offset": 10
        },
        {
          "key": 2,
          "length": 10,
          "offset": 18
        },
        {
          "key": 3,
          "length": 30,
          "offset": 33
        },
        {
          "key": 4,
          "length": 65,
          "offset": 68
        }
      ],
      "inlineStyleRanges": [
        {
          "length": 17,
          "offset": 0,
          "style": "UNDERLINE"
        },
        {
          "length": 45,
          "offset": 18,
          "style": "UNDERLINE"
        },
        {
          "length": 65,
          "offset": 68,
          "style": "UNDERLINE"
        },
        {
          "length": 102,
          "offset": 219,
          "style": "UNDERLINE"
        },
        {
          "length": 78,
          "offset": 322,
          "style": "UNDERLINE"
        },
        {
          "length": 28,
          "offset": 401,
          "style": "UNDERLINE"
        },
        {
          "length": 5,
          "offset": 38,
          "style": "ITALIC"
        },
        {
          "length": 102,
          "offset": 219,
          "style": "ITALIC"
        },
        {
          "length": 78,
          "offset": 322,
          "style": "ITALIC"
        },
        {
          "length": 28,
          "offset": 401,
          "style": "ITALIC"
        },
        {
          "length": 5,
          "offset": 48,
          "style": "BOLD"
        },
        {
          "length": 10,
          "offset": 58,
          "style": "BOLD"
        },
        {
          "length": 100,
          "offset": 219,
          "style": "{\"FG\":\"color5\"}"
        },
        {
          "length": 2,
          "offset": 319,
          "style": "{\"FG\":\"color2\"}"
        },
        {
          "length": 78,
          "offset": 322,
          "style": "{\"FG\":\"color2\"}"
        },
        {
          "length": 149,
          "offset": 401,
          "style": "{\"FG\":\"color2\"}"
        }
      ],
      "key": 2,
      "text": "#LIVING THE DREAM\nLeverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. \nOrganically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.\nBring to the table win-win survival strategies to ensure proactive domination.\nAt the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. ",
      "type": "code-block"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 3,
      "text": "",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 4,
      "text": "",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [
        {
          "length": 76,
          "offset": 0,
          "style": "{\"FG\":\"color2\"}"
        }
      ],
      "key": 5,
      "text": "User generated content in real-time will have multiple touchpoints for offshoring.",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 6,
      "text": "Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. ",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 7,
      "text": "Override the digital divide with additional clickthroughs from DevOps.",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 8,
      "text": "Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.",
      "type": "unstyled"
    }
  ],
  "entityMap": {
    "0": {
      "data": {
        "rel": "nofollow",
        "target": "_blank",
        "url": "https://www.wix.com/"
      },
      "mutability": "MUTABLE",
      "type": "LINK"
    },
    "1": {
      "data": {
        "rel": "nofollow",
        "target": "_blank",
        "url": "https://www.sport5.co.il"
      },
      "mutability": "MUTABLE",
      "type": "LINK"
    },
    "2": {
      "data": {
        "rel": "nofollow",
        "target": "_blank",
        "url": "https://www.wix.com/"
      },
      "mutability": "MUTABLE",
      "type": "LINK"
    },
    "3": {
      "data": {
        "rel": "nofollow",
        "target": "_blank",
        "url": "https://www.sport5.co.il/"
      },
      "mutability": "MUTABLE",
      "type": "LINK"
    },
    "4": {
      "data": {
        "rel": "nofollow",
        "target": "_blank",
        "url": "https://www.one.co.il/"
      },
      "mutability": "MUTABLE",
      "type": "LINK"
    }
  }
};

exports[`text > allow to create lists #0`] =
{
  "blocks": [
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 0,
      "text": "Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. ",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 1,
      "text": "Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.",
      "type": "ordered-list-item"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 2,
      "text": "Bring to the table win-win survival strategies to ensure proactive domination.",
      "type": "ordered-list-item"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 3,
      "text": "At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. ",
      "type": "ordered-list-item"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 4,
      "text": "User generated content in real-time will have multiple touchpoints for offshoring.",
      "type": "unordered-list-item"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 5,
      "text": "Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. ",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 6,
      "text": "Override the digital divide with additional clickthroughs from DevOps.",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 7,
      "text": "Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.",
      "type": "unstyled"
    }
  ],
  "entityMap": {}
};
